### PR TITLE
Formatting with ruff

### PR DESCRIPTION
**Pull request by xoudini**
Adds ruff config for `hatch fmt`, so might want to wait for #1 to be merged first (can't target that branch from here).

### CODE REVIEW

**Comment by occamz** [comment]
Merged #1

**Review by occamz — APPROVED** [review]

**Comment by xoudini** [comment]
> Merged #1

Rebased, had to drop the identical commits due to squash, but no problemo.

File: src/docker_snapshot/container.py [hunk 1]
```diff
@@ -148,7 +151,11 @@ def sync(source_directory, destination_path):
         [
             "sh",
             "-c",
-            f"rsync -aAHX --delete --info=progress2 {source_directory}/ {destination_path}",
+            # TODO: this could probably be split on every whitespace character
+            (
+                "rsync -aAHX --delete --info=progress2 "
+                f"{source_directory}/ {destination_path}"
+            ),
```
Comment (xoudini) [inline src/docker_snapshot/container.py:158]: Note: had to interpolate this string (or the other option would've been to ignore this line), but I'm pretty sure every component of the string could be split on whitespace. Felt like keeping everything semantically identical for now though.

**Comment by occamz** [comment]
> Rebased, had to drop the identical commits due to squash, but no problemo.

Ah, is it less confusing if I just rebase & merge? I've gotten used to squashing lately, but it isn't a principle or anything.

**Comment by xoudini** [comment]
> > Rebased, had to drop the identical commits due to squash, but no problemo.
> 
> Ah, is it less confusing if I just rebase & merge? I've gotten used to squashing lately, but it isn't a principle or anything.

Up to you, I'm okay either way, interactive rebase works fine for me 🙂

**Comment by occamz** [comment]
Allright, squashing to stay consistent (at least for today) :D

**Pull request merged**